File: python_backend/game_logic/misc/remove_captured.py
def remove_captured(init_ranges, pins, mt_restricts, final_ranges, captured):
    """remove the records of any piece that was captured"""
    if captured != 'None':
        del init_ranges[captured]
        del pins[captured]
        del mt_restricts[captured]
        del final_ranges[captured]
    return init_ranges, pins, mt_restricts, final_ranges


if __name__ == "__main__":
    pass # TODO: implement test

File: app (1)/src/game_logic/misc/removeCaptured.js


export function removeCaptured(initRanges, pins, mtRestricts, finalRanges, captured) {
    /*remove the records of any piece that was captured**/
    if (captured != 'undefined') {
        delete initRanges[captured]
        delete pins[captured]
        delete mtRestricts[captured]
        delete finalRanges[captured]
    }
    return [initRanges, pins, mtRestricts, finalRanges]
}

// module.exports = removeCaptured;

/** for node.js
if (require.main === module) {
    //TODO: implement test
}
*/